src/zap/zapimport.cpp

```diff
@@ -1993,238 +2003,268 @@ DWORD ZapIndirectHelperThunk::SaveWorker(ZapWriter * pZapWriter)
 {
     ZapImage * pImage = ZapImage::GetImage(pZapWriter);
 
-    BYTE buffer[44];
-    BYTE * p = buffer;
-
-#if defined(_TARGET_X86_)
-    if (IsDelayLoadHelper())
+#ifdef FEATURE_READYTORUN_COMPILER
```

Comment:
Unfortunate diff formatting. My change was just to wrap this section in an ifdef and an if(IsReadyToRunCompilation()). I then added a case for the runtime helper calls, which I've pointed out in the comment below.

Reply:
github protip: Add `?w=1` to the URL - it will ignore whitespaces and makes this diff easy to read.

Reply:
Neat, thanks!